missilewars-plugin/src/main/java/de/butzlabben/missilewars/util/PlayerDataProvider.java

```diff
@@ -21,66 +21,66 @@
 import de.butzlabben.missilewars.Logger;
 import de.butzlabben.missilewars.MissileWars;
 import de.butzlabben.missilewars.wrapper.player.PlayerData;
+import org.bukkit.entity.Player;
+
 import java.io.File;
 import java.util.HashMap;
 import java.util.UUID;
-import org.bukkit.entity.Player;
 
 public class PlayerDataProvider {
 
-    private static final PlayerDataProvider ourInstance = new PlayerDataProvider();
-
-    private final String path = "data";
+    private static final PlayerDataProvider instance = new PlayerDataProvider();
     private final HashMap<UUID, PlayerData> data = new HashMap<>();
+    private final File playerDataFile;
```

Comment:
```suggestion
    private final File playerDataDirectory;
```
Rename to this, as file would mean a single file, which it is in fact not

Reply:
I did. Alternatively, I can create a new PlayerData object where the constructor stores the default values before applying it with the same method.